### ISSUE TITLE
Main command menu items multiply during remote control switch 

### ISSUE THREAD

**Issue by nirtulys** [Wyqer/kp_liberation#142]
### Basic Information
**Mission version:**
0.953

**Map used:**
Altis

**Mods used:**
CBA, ace, Advanced AI Command, ShackTack UI

**Dedicated or local game?**
Dedicated

**GitHub or Workshop mission file?**
GitHub & Workshop

### Individual Things
**Did you edit anything within the mission files?**
No

### Short Issue Description
Main command menu items multiply during remote control switch 

### Expected Behaviour
Menu item count remain the same.

### Steps to Reproduce the Issue
In game:
- create unit, enter map, select unit,
- from unit menu select remote control,
- end remote control with "del" key,
- check main command menu item count. 

### Screenshots, Serverlog or any other helpful Information
https://steamuserimages-a.akamaihd.net/ugc/844836780024694329/99D993495CB5E6751B123453F024D052552C417D/

**Comment by Wyqer** [comment]
Oh that's really interesting. Thanks for the report. I'll look into that and fix it in 954. Again many thanks for the report.

**Comment by amunoz20** [comment]
I Just wanna point out that this problem is caused by the Advanced AI Command. When you remote control a unit the game thinks that now that unit is the commander so now you have 2 commanders in game. If the Squad that you remote control has múltiple units or you remote control several units, this behaviour will keep in going. 

Also when you remote control a unit, your previous unit (the original) becomes like an AI and sometimes it recibes orders from something which causes that your original unit moves and do things while you are remote control inglés. 

To recap, the problem is caused by Advanced AI Command and this may very well be something non related to liberation itself. @wyqer Its worth investigating because if you fix it you will solve my number one problem of every day hahaha

**Comment by amunoz20** [comment]
@nirtulys just to prove a point are you playing alone (lone wolf)? have you tired to change your permissions in game to let the commander be the only one Who can build etc etc..? If you change your permissions for commander only will the error keep on happening?

**Comment by nirtulys** [comment]
Good question. I have permissions disabled. Yes, I'm playing solo.

And while I agree, that this issue is caused by AIC. Although I used this mod in various other missions and never had such issue. To me it looks like menu item list is not updating correctly while changing player.

**Comment by Wyqer** [comment]
Oh, that's an important hint. I've thought you used the liberation inbuilt group management for that.
Never used AIC, so I can't really say what may be wrong with it.
But I'll have a look in the action manager if I may implement anything which would prevent something like that.

**Comment by amunoz20** [comment]
@nirtulys like I said, this problem May be caused by AIC related with liberation. Im not in front of my gaming rig atm so I cant test this but if you can just check what I said about the permissions. This would give Wyqer a very interesting hint to maybe block the problem. However this is not gonna be the first problem you encounter. I am also a solo player who uses AIC and C2, regarding to AIC you can expect missbehaving of UAVs, AI units getting stuck while rappeling or free falling from helis.. etc.. Feel free to add me on steam and discord as Jettfiremachine, Im not part of the de team nor part of the support group but i have plenty of experience playing liberation solo since version 0.924 ;)

**Comment by amunoz20** [comment]
@wyqer still dont quote me on that.. I may be super wrong. This theory of multiple commanders is just a hunch. It could be tested with the permissions manager
